docs/course_materials/exercises/01_Linux/class_exercise_4.md

```diff
@@ -0,0 +1,11 @@
+### Let's Practice Together \#4
+
+- Open a terminal
+- Create a Directory **linux4** in you home directory
```

Comment:
home > **home**

Reply:
done

Reply:
it still seems to be home and not **home**